.github/actions/create-default-e2e-config-override/action.yml

```diff
@@ -0,0 +1,149 @@
+        fi
+        if [ -n "${{ inputs.chainlink_upgrade_image }}" ]; then
+          create_args+=(--chainlink-upgrade-image="${{ inputs.chainlink_upgrade_image }}")
+        fi
+        if [ -n "${{ inputs.chainlink_upgrade_version }}" ]; then
+          create_args+=(--chainlink-upgrade-version="${{ inputs.chainlink_upgrade_version }}")
+        fi
+        if [ -n "${{ inputs.pyroscope_server }}" ]; then
+          create_args+=(--pyroscope-server-url="${{ inputs.pyroscope_server }}")
+        fi
+        if [ -n "${{ inputs.pyroscope_environment }}" ]; then
+          create_args+=(--pyroscope-environment="${{ inputs.pyroscope_environment }}")
+        fi
+        if [ -n "${{ inputs.pyroscope_key }}" ]; then
+          create_args+=(--pyroscope-key="${{ inputs.pyroscope_key }}")
+        fi
+        if [ -n "${{ inputs.loki_endpoint }}" ]; then
+          create_args+=(--logging-loki-endpoint="${{ inputs.loki_endpoint }}")
+        fi
+        if [ -n "${{ inputs.loki_tenant_id }}" ]; then
+          create_args+=(--logging-loki-tenant-id="${{ inputs.loki_tenant_id }}")
+        fi
+        if [ -n "${{ inputs.loki_basic_auth }}" ]; then
+          create_args+=(--logging-loki-basic-auth="${{ inputs.loki_basic_auth }}")
+        fi
+        if [ -n "${{ inputs.loki_run_id }}" ]; then
+          create_args+=(--logging-run-id="${{ inputs.loki_run_id }}")
+        fi
+        if [ -n "${{ inputs.test_log_collect }}" ]; then
+          create_args+=(--logging-test-log-collect="${{ inputs.test_log_collect }}")
+        fi
+        if [ -n "${{ inputs.private_ethereum_network_execution_layer }}" ]; then
+          create_args+=(--private-ethereum-network-execution-layer="${{ inputs.private_ethereum_network_execution_layer }}")
+        fi
+        if [ -n "${{ inputs.private_ethereum_network_custom_docker_image }}" ]; then
+          create_args+=(--private-ethereum-network-custom-docker-image="${{ inputs.private_ethereum_network_custom_docker_image }}")
+        fi
+        
+        # # Split the log targets input by comma and add to the command line arguments
+        IFS=',' read -ra ADDR <<< "${{ inputs.logstream_log_targets }}"
+        for i in "${ADDR[@]}"; do
+          create_args+=(--logging-log-targets="$i")
+        done
+
+        # Split selected_networks input by comma and add to the command line arguments
+        IFS=',' read -ra ADDR <<< "${{ inputs.selected_networks }}"
+        for i in "${ADDR[@]}"; do
+          create_args+=(--selected-networks="$i")
+        done
+
+        config_override=$(go run main.go test-config create "${create_args[@]}")
+        # echo "go run main.go test-config create ${create_args[*]}"
+        # echo "Output:"
+        # echo "$config_override"
+
+        BASE64_CONFIG_OVERRIDE=$(echo "$config_override" | base64 -w 0)
+        echo ::add-mask::$BASE64_CONFIG_OVERRIDE
+        echo "BASE64_CONFIG_OVERRIDE=$BASE64_CONFIG_OVERRIDE" >> $GITHUB_ENV        
```

Comment:
can we do all/some of these operations inside the tool? pretty sure we could add a `--base64-output` flag to do `base64 -w 0`.
maybe we could also call `echo ::add-mask::$BASE64_CONFIG_OVERRIDE` from go? or even send that to `$GITHUB_ENV`? (it's a file after all)